docs/api/v3.rst

```diff
@@ -0,0 +1,558 @@
+                {
+                    "href": "/api/v3/project/pip/notifications/",
+                    "rel": "notifications",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/features/",
+                    "rel": "features",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/subprojects/",
+                    "rel": "subprojects",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/translations/",
+                    "rel": "translations",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/integrations/",
+                    "rel": "translations",
+                    "type": "GET"
+                }
+            ]
+        }
+
+
+    :>json string name: The name of the project.
+    :>json string slug: The project slug (used in the URL).
+    :>json string description: An RST description of the project
+    :>json string language: The language code of this project
+    :>json string programming_language: The programming language of the project (eg. "py", "js")
+    :>json string repo: The repository URL for the project
+    :>json string repo_type: Version control repository of the project
+    :>json string default_version: The default version of the project (eg. "latest", "stable", "v3")
+    :>json string default_branch: The default version control branch
+    :>json string documentation_type: An RST description of the project
+    :>json string canonical_url: The canonical URL of the default docs
+    :>json array links: Array with HEATEOAS_ links to retrieve related information
+
+    :statuscode 200: Success
+    :statuscode 404: There is no ``Project`` with this slug
+
+.. TODO:
+
+   Currently, v2 of this endpoint returns a lot of fields more like
+   ``enable_epub_build``, ``skip``, etc.
+
+   https://readthedocs.org/api/v2/project/?slug=pip
+
+
+Project import
+++++++++++++++
+
+.. http:post::  /api/v3/project/import/
+
+    Import a new project.
+
+    **Example request**:
+
+    .. sourcecode:: js
+
+        {
+            "name": "Pip",
+            "repo_url": "https://github.com/pypa/pip",
+            "repo_type": "git"
+            "description": "Pip Installs Packages.",
+            "documentation_type": "sphinx_htmldir",
+            "language": "en",
+            "programing_language": "py",
+            "tags": [
+                "pip",
+                "python",
+                "packaging"
+            ],
+            "homepage": "https://pip.readthedocs.io/"
+        }
+
+    ``name``, ``repo_url`` and ``repo_type`` are required.
+
+    **Example response**:
+
+    *See Project details*
+
+    :statuscode 201: Created sucessfully
+    :statuscode 400: Some field is invalid
+    :statuscode 401: Not valid permissions
```

Comment:
I don't think this is necessary to list on every endpoint. This should probably just be in some section on authentication.